soh/soh/Enhancements/game-interactor/GameInteractor_RawAction.cpp

```diff
@@ -103,7 +103,9 @@ void GameInteractor::RawAction::ForceEquipBoots(int8_t boots) {
 }
 
 void GameInteractor::RawAction::FreezePlayer() {
-    gSaveContext.pendingIceTrapCount++;
+    Player* player = GET_PLAYER(gPlayState);
+    player->actor.colChkInfo.damage = 0;
+    func_80837C0C(gPlayState, player, 3, 0, 0, 0, 0);
```

Comment:
This change was required because the ice traps can now be processed by game interactor, so this caused a infinite loop of ice traps. Not sure if there is a better way to handle this..